src/Build/Evaluation/PropertyTrackingEvaluatorDataWrapper.cs

```diff
@@ -373,20 +388,32 @@ private enum PropertySource
             BuiltIn,
             Global,
             Toolset,
-            EnvironmentVariable
+            EnvironmentVariable,
+            CommandLine,
```

Comment:
What's the difference between Global and CommandLine? What do we log for API uses that set global properties?

Reply:
We log "Global" for API uses
https://github.com/dotnet/msbuild/blob/7e539c169a657040eeb768e8115c44f1780e59df/src/Build/Evaluation/Evaluator.cs#L1254

Reply:
Currently it's labeled from the script:

![{16B6CB47-8AAE-4D28-9B56-504B922D7EDA}](https://github.com/user-attachments/assets/26c33153-08bf-438a-b04b-c4a1a9dded7d)

Reply:
![{B137F366-BA94-4EF6-A99A-DAB0134C0F13}](https://github.com/user-attachments/assets/1af55293-bae9-45ab-89d9-c46ddffe219a)
here you go